modules/nw-ovn-kuberentes-limitations.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc
+
+[id="nw-ovn-kubernetes-limitations_{context}"]
+= OVN-Kubernetes limitations
+
+The OVN-Kubernetes default Container Network Interface (CNI) network provider has a limitation that is related to traffic policies.
```

Comment:
This is my fault from being overly technical; While it is the `default` plug-in for Multus, I think literally there's a `default` key that is specified, in practice this can be confusing, so I think Marc Curry suggested I use _cluster network provider_ instead, or _CNI cluster network provider_, but omit the word _default_. I haven't changed this everywhere in the docs yet, though.

```suggestion
The OVN-Kubernetes Container Network Interface (CNI) cluster network provider has a limitation that is related to traffic policies.
```